iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/extensions/DiscreteMeasurementsXmlSerializer.java

```diff
@@ -107,18 +107,20 @@ public void write(DiscreteMeasurements<I> extension, XmlWriterContext context) t
             adder.setTapChanger(DiscreteMeasurement.TapChanger.valueOf(tapChanger));
         }
         DiscreteMeasurement.ValueType valueType = DiscreteMeasurement.ValueType.valueOf(reader.getAttributeValue(null, "valueType"));
```

Comment:
I think you should check before if `reader.getAttributeValue(null, "valueType")` is not null, otherwise `valueType` is required.

Reply:
`valueType` is  required, so I don't think we need to check if it is `null`?

Reply:
Ah ok, got it! I thought that if value was optional, so should be valueType